src/shims.h

```diff
@@ -41,6 +41,8 @@
 #if HAVE_PTHREAD_WORKQUEUES
 #if __has_include(<pthread/workqueue_private.h>)
 #include <pthread/workqueue_private.h>
+#elif DISPATCH_USE_INTERNAL_WORKQUEUE
+#include "event/workqueue_internal.h"
```

Comment:
duh, I had resolved the merge conflict the wrong way around, thanks for spotting

squashed the fix